src/locales/nl/concerts.yaml

```diff
@@ -356,3 +356,15 @@ concerts:
       Benieuwd? Zaterdag 14 maart speelt het Krashna KamerOrkest in de Lutherse Kerk Delft om 20:15 uur! Kaartjes zijn €13 (regulier) en €8 (jonger dan 25 jaar) aan de deur. Reserveer vooraf via [krashko@krashna.nl](mailto:krashko@krashna.nl) en ontvang 50ct korting per kaartje.
 
       ![Digital Flyer](/images/krashko-2020-digiflyer.png?style=centerme-75)
+
+  livestream-kamu-august-2020:
+    title: Krashna Kamermuziekconcert Livestream
+    description: |
+      De afgelopen maanden heeft Krashna door de maatregelen rondom covid-19 veel evenementen af moeten zeggen. Om dit collegejaar toch nog een leuk einde te geven, heeft de kamermuziekafdeling van Krashna Musika een heel speciaal concert geörganiseerd, namelijk: een live uitgezonden kamermuziekconcert, dat vanaf youtube bekeken kan worden.
+
+      Op bold(zondag 2 augustus om half 3 's middags) zullen wij van start gaan. Muzikanten zullen live muziek maken op twee locaties. Het wordt professioneel gefilmd met medewerking van de livestream commissie van roeivereniging Proteus Eretes, waardoor u het vanuit uw eigen huis kunt beluisteren en bekijken
```

Comment:
The bold thing, Proteus-Eretes is with hyphen. Please end the sentence with a dot.